common/lib/xmodule/xmodule/modulestore/mongo/base.py

```diff
@@ -1905,7 +1905,7 @@ def ensure_indexes(self):
         self.collection.create_index('definition.children', sparse=True)
 
         # To allow prioritizing draft vs published material
-        self.collection.create_index('_id.revision')
```

Comment:
I think we should have a method that is used/usable for production. It seems like if we have that, then local use can just use that as well.